ilastik/applets/tracking/structured/structuredTrackingGui.py

```diff
@@ -107,7 +105,14 @@ def _loadUiFile(self):
             self._drawer.appearanceBox.setValue(parameters['appearanceCost'])
         if 'disappearanceCost' in parameters.keys():
             self._drawer.disappearanceBox.setValue(parameters['disappearanceCost'])
-        
+
+        solverName = self.topLevelOperatorView._solver
```

Comment:
Side note: Did you already provide a solver selection drop down box as in the normal tracking workflow?